### DISCUSSION TITLE
Add user

### DISCUSSION THREAD

**Discussion by AbsentGalaxy** [anselal/antminer-monitor#198] [Q&A]
Is there anyway to add more users

**Comment by anselal** [comment]
Yes, since we use a database to store users you can add more users. But the app currently does not support roles so there won't be basically a difference between the users. They still will have admin rights.

**Reply by AbsentGalaxy** [reply]
Thank you, I was wanting to set up multiple users so they can monitor individual miners that they are assigned to.

**Reply by anselal** [reply]
this can be done, but is currently not supported.
I will need to create groups of miners, then add the ability to add more users, have roles, and the admin would be able to assign multiple groups to users or roles.
This is a nice feature to be honest.
I don't have access to any miners at the moment so even if I would find the time to implement it, it would be without testing it first...

**Reply by Spiritualgangbanger** [reply]
🙏👌🙈⚕️
